.github/workflows/update-docs-version.yml

```diff
@@ -0,0 +1,30 @@
+name: Update docs version
+
+on:
+    release:
+        types: [published]
+
+jobs:
+    build:
+        runs-on: ubuntu-18.04
+        steps:
+            -   uses: actions/checkout@v1
+            -   name: Update latest_version property in mkdocs.yml
+                run: |
+                    sed -i "s/latest_version: .*/latest_version: ${GITHUB_REF##*/}/g" mkdocs.yml
```

Comment:
1) this should go after `git checkout master`
2) idea: add `git diff` after `sed` for debugging